src/tests/mem/page_allocator.c

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @date 12.07.2018
+ * @author Filipp Chubukov
+ */
+
+#include <embox/test.h>
+#include <mem/page.h>
+#include <mem/heap.h>
+#include <mem/phymem.h>
+#include <util/binalign.h>
+
+EMBOX_TEST_SUITE("page_alloc_init test");
+
+TEST_CASE("Init page_allocator with one page") {
```

Comment:
I do not see serios differences between these two test cases. As I understand you are testing two cases: len == page_size and len < page_size. In both cases, you should get NULL. So you just can propose united test case for len <= page_size and test both variants inside it.

United variant:

allocator =  page_allocator_init(buff, 0x100, 0x100);
test_assert_null(allocator);

allocator =  page_allocator_init(buff, 0x10, 0x100);
test_assert_null(allocator);

Reply:
But it is not critical, so we can leave as it is.